docs/interpreter/livy.md

```diff
@@ -61,6 +61,11 @@ Example: `spark.driver.memory` to `livy.spark.driver.memory`
     <td>Max number of Spark SQL result to display.</td>
   </tr>
   <tr>
+    <td>zeppelin.livy.spark.sql.truncate</td>
+    <td>true</td>
+    <td>Whether to truncate strings or not</td>
```

Comment:
I wrote the description to match the documentation of the corresponding truncate argument in the spark dataset method
[http://spark.apache.org/docs/latest/api/scala/index.html#org.apache.spark.sql.Dataset@show(numRows:Int,truncate:Boolean):Unit](http://spark.apache.org/docs/latest/api/scala/index.html#org.apache.spark.sql.Dataset@show(numRows:Int,truncate:Boolean):Unit)

> truncate -
> Whether truncate long strings. If true, strings more than 20 characters will be truncated and all cells will be aligned right

This should be applicable only to the LivySparkSQLInterpreter. Only in this interpreter,  we invoke the above method as part of the code. For other interpreters, the user supplied code can call the appropriate show() method.